### DISCUSSION TITLE
Does this work on forge?

### DISCUSSION THREAD

**Discussion by ChesterProgram** [sladkoff/minecraft-prometheus-exporter#149] [Q&A]
The title says it all.

**Comment by Always-Self-Hosted** [comment]
Can this be answered please? I'm keen to use this if possible for my [stone block 4 server](https://www.curseforge.com/minecraft/modpacks/ftb-stoneblock-4). netdata [explicitly mentions using this project](https://www.netdata.cloud/monitoring-101/minecraft-monitoring/) with it.
